app/jobs/unlock_teacher_account_job.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+class UnlockTeacherAccountJob < ApplicationJob
```

Comment:
Now that we have this, we can replace the call to `DqtApi.unlock_teacher!(teacher_account)` with `UnlockTeacherAccountJob.perform_later(teacher_account)` in `DqtApi`; this will make the `find_trn!` action quicker, as it will make the unlock teacher action run asynchronously :+1: